make/modules/java.desktop/lib/AwtLibraries.gmk

```diff
@@ -273,7 +256,6 @@ ifeq ($(call isTargetOs, windows macosx)+$(ENABLE_HEADLESS_ONLY), false+false)
       LIBS_unix := $(LIBDL) $(LIBM) $(X_LIBS) -lX11 -lXext -lXi -lXrender \
           -lXtst, \
```

Comment:
Same question as for the STATIC_LIB_EXCLUDE_OBJS change with `libjli`. Are the duplicate symbol issues resolved by symbol hiding?

I think it's still better to not include those .o files to avoid unnecessary footprint overhead in the binary.

Reply:
> Are the duplicate symbol issues resolved by symbol hiding?

Yes, correct.

> I think it's still better to not include those .o files to avoid unnecessary footprint overhead in the binary.

This was introduced as a being completely necessary to produce static builds, not as a means to reduce binary size. These kind of hard-coded lists are always a maintenance problem, and I do not think they are worth keeping just for the sake of reducing binary size.

If that is an issue, we can probably add some LTO step when doing the partial link. I'd guess that will be much more beneficial in terms of size reduction that to exclude a handful of manually picked files.

Having this kind of excludes furthermore makes it harder to selectively only include libraries belonging to a specific subset of modules, since it means that some libraries will not work unless other -- unspecified -- libraries are also included in the link.